nemo_curator/classifiers/__init__.py

```diff
@@ -18,11 +18,13 @@
 from .aegis import AegisClassifier
 from .domain import DomainClassifier
 from .fineweb_edu import FineWebEduClassifier
+from .pytorch_deberta import PyTorchClassifier
```

Comment:
I dont think we should add it to a cli. We should just add a tutorial around it.